cms/templates/js/publish-xblock.underscore

```diff
@@ -86,7 +93,8 @@ var visibleToStaffOnly = visibilityState === 'staff_only';
                 <% } %>
                 <%= gettext('Hide from students') %>
             </a>
-        </p>
+            </li>
```

Comment:
I don't see any accessible text here, just an icon font (inaccessible).  Is it added via JS?  If not, we need accessible text here, otherwise this button has no accessible label.  nit: add a ```#``` to the href